op-program/client/l2/engine.go

```diff
@@ -17,7 +17,12 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
-var ErrNotFound = errors.New("not found")
+var (
+	ErrNotFound              = errors.New("not found")
```

Comment:
I wonder if this should be `ethereum.NotFound`? Unlikely to actually matter but it is very commonly used for this kind of situation and maybe makes sense to reuse here.

Reply:
it should be. It's the exact same error and it can replace the 5 different "not found" error definitions in the program. I'll adjust.